### PR TITLE
Section on sponsoring

### PR DESCRIPTION
**Pull request by JesperIRL**
Copied and updated the text from the old sponsoring page: https://openjdk.org/sponsor/
The purpose is to have the sponsor link on the OpenJDK page lead here instead.

Also updated the contributing section with some parts taken from the old contributing page: https://openjdk.org/contribute/
Again, the intention is to change the link on the OpenJDK page to lead here.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**) ⚠️ Review applies to [851c658c](https://git.openjdk.org/guide/pull/97/files/851c658c7866598b2406366ca15f923558e595f0)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/guide.git pull/97/head:pull/97` \
`$ git checkout pull/97`

Update a local copy of the PR: \
`$ git checkout pull/97` \
`$ git pull https://git.openjdk.org/guide.git pull/97/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 97`

View PR using the GUI difftool: \
`$ git pr show -t 97`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/guide/pull/97.diff">https://git.openjdk.org/guide/pull/97.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/guide/pull/97#issuecomment-1412365339)

### CODE REVIEW

File: src/guide/contributing-to-an-open-jdk-project.md [hunk 1]
```diff
@@ -22,13 +22,19 @@ In many GitHub projects the standard way to propose a change is to create a pull
...
 
 When you sign the OCA, please make sure that you specify your GitHub user name in the `Username` field of the OCA. If you try to create a PR before you have signed the OCA, or if you didn't specify your GitHub user name, you'll get instructions telling you to do so, and the PR won't be published until this is done. OCA registration is a manual process. Please allow for up to several days to have your OCA application processed, even though it's normally processed swiftly. An alphabetical list of all of the assigned OpenJDK usernames may be found on the [OpenJDK people](https://db.openjdk.org/people) list.
 
+You only need to sign the OCA once in order to cover all changes that you might contribute to any Oracle-sponsored open-source community. If you've already signed the OCA or the former SCA (Sun Contributor Agreement) for any Oracle-sponsored open-source community, then you do not need to sign it again in order to contribute to OpenJDK. Please note that you don't need to sign an OCA if you work at Oracle or a company which has negotiated an equivalent agreement.
```
Comment (dholmes-ora) [inline src/guide/contributing-to-an-open-jdk-project.md:29]: Not sure if this needs to clarify signing as an individual versus being covered by an employers OCA. But the probably needs to be stated somewhere in the first paragraph:

"If you have not signed the OCA and are not an employee of a Company that has an OCA covering you, then ..."

File: src/guide/contributing-to-an-open-jdk-project.md [hunk 2]
```diff
@@ -52,7 +58,9 @@ Even though we strive to unify how things are done within OpenJDK, different are
 * [CSR Process](https://wiki.openjdk.org/display/csr/Main)
 :::
 
-Java and the JDK are very popular products, and just about every Java developer out there has an idea or two for how to enhance something. And (obviously not referring to you) believe it or not, not every idea is a good idea. Even though many ideas are indeed good, we must be quite restrictive on what we actually include into the JDK. There are many reasons for this.
+Only the best patches submitted will actually make it all the way into the JDK code base. The goal is not to take in the maximum number of contributions possible, but rather to accept only the highest-quality contributions. The JDK is used daily by millions of people and thousands of businesses, often in mission-critical applications, and so we can't afford to accept anything less than the very best.
```
Comment (dholmes-ora) [inline src/guide/contributing-to-an-open-jdk-project.md:61]: "Only the best" is not really a defining criteria here. This paragraph really overlaps with the following paragraph. First the idea must be appropriate and agreed that a patch is needed, then the quality of the patch itself will be assessed using various criteria depending on the exact nature of the patch.

File: src/guide/sponsoring-a-change.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Sponsoring a Change
+
+New developers in the OpenJDK community doesn't have the permissions needed to push changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get the first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually push the sponsored contribution into the repository.
```
Comment (dholmes-ora) [inline src/guide/sponsoring-a-change.md:3]: s/doesn't/don't/ in first sentence.

This definition of sponsor seems to be from the old days when you basically needed someone from Sun/Oracle to help shepherd your change through. These days any of the reviewers will offer advice and may become the eventual sponsor of the commit. So I don't think this section really applies any more i.e. sponsoring-a-change.md doesn't need to exist.

File: src/guide/sponsoring-a-change.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+There are many different reasons to sponsor a change and depending on the situation the exact steps involved may differ significantly. A developer hired into a larger team of experienced OpenJDK developers is likely to get the required information in new hire training or similar and will likely need less help from a sponsor than the entusiast sitting at home with no prior experience of large scale software development. This text aims to highlight the steps involved in sponsoring a change but can't cover all possible scenarios. As always, common sense takes precedence where the assumptions made here doesn't apply to your use case.
```
Comment (dholmes-ora) [inline src/guide/sponsoring-a-change.md:5]: typo: entusiast

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 1 above)
Comment (JesperIRL) [reply src/guide/contributing-to-an-open-jdk-project.md:29]: Isn't that already covered by the last sentence?
"Please note that you don't need to sign an OCA if you work at Oracle or a company which has negotiated an equivalent agreement."

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 2 above)
Comment (JesperIRL) [reply src/guide/contributing-to-an-open-jdk-project.md]: I rephrased it somewhat.

File: src/guide/sponsoring-a-change.md (hunk 3 above)
Comment (JesperIRL) [reply src/guide/sponsoring-a-change.md]: I fixed the typo.

It is indeed a text from the old days. Sponsoring is still a thing though so we should have some description of what it means and what to expect from a sponsor. It's a fairly short section, and it may be that some of it can be cut or rephrased to better suit the present. If it becomes much shorter it may fit better as a sub-section under "Contributing to an OpenJDK Project".
Shortening this text is not at the top of my priority list right now though. I just want to get something that is good enough to change the "Sponsoring" link in the OpenJDK sidebar to refer to the guide. That said, if someone else takes a stab at making this text even more up to date I'd be happy to review and sponsor that change :-)

File: src/guide/sponsoring-a-change.md (hunk 4 above)
Comment (JesperIRL) [reply src/guide/sponsoring-a-change.md]: Fixed.

**Comment by JesperIRL** [comment]
Thank you for reviewing @dholmes-ora!

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 1 above)
Comment (dholmes-ora) [reply src/guide/contributing-to-an-open-jdk-project.md:29]: Yes but by the time you get to that last sentence (if you read that far) you've already been told numerous times you should sign the OCA. I suggest in the first paragraph that this:

> and if you haven't yet signed the [Oracle Contributor Agreement](https://oca.opensource.oracle.com/) (OCA) then please do so

is expanded to:

> and if you haven't yet signed the [Oracle Contributor Agreement](https://oca.opensource.oracle.com/) (OCA), and are not covered by a company-level agreement, then please do so

File: src/guide/sponsoring-a-change.md (hunk 3 above)
Comment (dholmes-ora) [reply src/guide/sponsoring-a-change.md]: The point is that "sponsor" is merely  some committer that now types /sponsor in your PR. The days when you needed to find a sponsor upfront and they needed to shepherd your change through the system (and run testing etc) are long gone.

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 1 above)
Comment (JesperIRL) [reply src/guide/contributing-to-an-open-jdk-project.md:29]: Fixed.

File: src/guide/sponsoring-a-change.md (hunk 3 above)
Comment (JesperIRL) [reply src/guide/sponsoring-a-change.md]: Is that really the case? You would still need someone to run proper testing on multiple platforms etc, so even though the actual integration is easier today the work up to the integration point is basically the same, isn't it?

Comment (dholmes-ora) [reply src/guide/sponsoring-a-change.md]: If you can't test a change adequately yourself and need some help, that is true whether you are an Author or Committer, or even Reviewer. Test help is not an integral part of sponsorship.

**Comment by JesperIRL** [comment]
Reworked the entire sponsoring section into a "Review and Sponsor"-section. Hopefully all concerns regarding the old text is addressed now. Hopefully the text is also intact after me trying to bring in changes from the master branch... It looks good as far as I can see.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [GitHub Docs on pull requests reviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews)
+:::
+
+Two of the most important contributions you can make in the OpenJDK community are to review and sponsor changes for other developers. All changes needs to be reviewed. Many changes even need to be reviewed by more than one person. This means that in order to enable a fast-pased development experience, all developers need to review more changes then they produce themself.
+
+If you are new to an area, reviewing changes is also a great way to learn the code and see what general styles and types of changes that are relevant in the area. Be mindful though, if you don't know the area well you should state this in your review comments. As a **R**eviewer you have a responsibility to make sure changes are sound and goes in line with the general direction of the area. If you, as a **R**eviever, review a change in an area that you don't know well you probably shouldn't be the one to approve the change.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:11]: s/changes that/that/
s/goes in line/go in line/ or perhaps better just "and align with ..."

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+New developers in the OpenJDK community don't have the permissions needed to push changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get the first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually push the sponsored contribution into the repository.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:13]: s/get the first/get their first/
s/eventually push/eventually integrate/ (the developer does the git push)

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+Sponsoring new contributions is an important activity - it's how the engineering culture of a project gets passed on from the core group to new Contributors, from one generation to the next. It should be fun, so please celebrate the contributions you've sponsored by mentioning them on your blog or similar. Congratulate other Sponsors on their work. Take pride in the value you provide to Contributors. Their success reflects well on you.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:15]: This paragraph is a bit over-the-top for me. I would personally drop the "It should fun ..." and "Congratulate ..." sentences.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+As a Sponsor, Contributors will look up to you for guidance to get their contributions into the project - your actions will determine whether Contributors will feel welcome and want to engage further with the project beyond their initial attempt, or not. Let's not lose enthusiastic, engaged and technically competent Contributors due to a lack of communication. If there is a request in your area of expertise and you can't address it, at least acknowledge receipt of the request and provide an estimate for when you'll be able to give it your attention. A frank explanation of your time constraints or commitments will be appreciated and respected.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:17]: This, to me, makes being a Sponsor sound like an active role i.e. you sit and watch all day for things that your might Sponsor; or there is a notion of a designated sponsor for an area who may not have time for something at the moment. But normally it is not like that. If a change has been socialized ahead of time then a sponsor tends to emerge from the participants in the PR. If a change has not been socialized and a PR just appears then it may well just sit there because it isn't on anyone's radar - not even to say "I'm too busy for this" because they are too busy to pay it any attention in the first place.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+## Reviewing a change on GitHub
+
+Code reviews in OpenJDK happens on [GitHub](https://github.com/openjdk). This guide will hint at the most common steps to perform a review but it's not a complete documentation of all features. For official documentation see the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+
+When you arrive at a pull request there are a few different sections that you can browse through, Conversation, Commits, Checks, and Files changed. We will look at the first and the last of these.
+
+### Conversation
+
+The Conversation section is where all discussion around the PR happens. Note that this section is intended for technical discussion around the actual implementation of a change. This is not the place to discuss the appropriateness of the change or have overarching design discussions, that should happen on the appropriate [mailing lists].
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:29]: I have to disagree about the last part - these days because the PR comments go to the mailing list and mailing-list-only comments (in a PR thread) go to the PR, it is indeed the case that design discussions and questions of appropriateness appear in the PR. Personally I prefer JBS to capture such things but the PR has become the one-place-shop in many cases and JBS commentary is typically very limited. Also JBS can't be used by a developer who needs a sponsor and doesn't yet have Author role.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+You can either reply to existing comments by using the text field beneath each comment, or add a new comment by scrolling to the bottom of the page and use the larger text field where it says "Leave a comment".
+
+Reviews take time for everyone involved. Your opinions are much appreciated as long as they are on-topic and of a constructive nature. A comment of the form "I don't like this" won't help anyone to improve the change. Instead express what it is that you don't like and give exmaples of how to do it differently. Always be polite and make sure you read through all previous comments before adding your own, the answer you are looking for may already have been given.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:33]: Typo: exmaples

I would add a note to check for hidden comments and resolved issues, as they may contain information relevant to your own comment

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+Never ask the author of a change to fix unrelated issues in the code that you find during your review. Such issues should be handled through a separate JBS issue and PR.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:35]: Yes and no. :) Unrelated functional fixes are generally not encouraged. Sometimes people will request style or similar fixups "while you are there". On the other hand, many reviewers will object to large scale code cleanups that are not essential to the actual fix and which obscure it. So it is a fine line here.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+### Files changed
+
+The Files changed-section is where the actual change is presented. On the left hand side there's a list of all files changed. This is useful to get a quick overview and to see if the change touches an area that you need to review.
+
+The diff view is pretty straight forward, red stuff has been removed, green stuff has been added. You can configure (among other things) if you want a unified diff or a split view, if you want to hide whitespace changes, what types of files you want to show, and if you want to show the entire change or just what's happened since your last review.
+
+When you hover the text in the change a blue square with a plus sign pops up to the left of the line. Click it to add a comment about the change on the corresponding line. When you create your first comment you have the option to simply add a single comment or to start a review. In general you want to start a review so that all your comments are gathered in the same context when emails are sent out etc. You can also attach a comment to a file rather than to a specific line in the file by using the speach bubble icon at the top right.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:43]: Typo: speach

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+## Responsibilities of a sponsor
+
+As a sponsor, you aren't technically required to review the change, other Reviewers may already have looked at it or signed up to review. But it will be your name on the commit so you should do whatever you feel is needed to be comfortable with that. You may need to work with the Contributor to make any necessary changes, until you and the Contributor are satisfied with the result, and you are satisfied that the proposed contribution will pass any relevant review and testing. That may take more than one iteration in the worst case. You may also need to help out with any process or practical questions around the OCA, GitHub PRs, and specific requirements in the area or project.
+
+To integrate the change the Contributor should use the command `/integrate` on the PR as prompted by the bots. Once that is done, you as a sponsor enters the command `/sponsor` in a comment on the PR.
```
Comment (dholmes-ora) [inline src/guide/reviewing-and-sponsoring-a-change.md:67]: s/enters/enter/

**Review by dholmes-ora — COMMENTED** [review]
Hi Jesper,

A few comments below - just my opinion so take them, or leave them, as you will. A few typos too.

Thanks.

**Comment by JesperIRL** [comment]
Thank you @dholmes-ora for your review! Much appreciated. I have made changes based on your comments. The only comment I don't know how to handle is the one about comments in PR vs mailing lists. As you point out, all comments made in a PR will be sent to the mailing list as well, so saying that one is preferred and the other is not seems weird. Keeping design discussions in JBS has it's challenges. In general in this guide we tend to refer to the mailing lists for all such discussions. I need more input from others to get that paragraph right.

File: src/guide/reviewing-and-sponsoring-a-change.md (hunk 9 above)
Comment (magicus) [reply src/guide/reviewing-and-sponsoring-a-change.md:29]: I think PRs are typically used to discuss the *implementation* once you have one. If someone opens a JBS issue with e.g. a new feature, but with no corresponding code, then the discussion will happen there. 

From time to time a PR is published with a concrete implementation, but then a discussion arises about the approriateness of the design, or even if the problem should be resolved at all etc. I don't think it is ideal if that kind of discussions happen on a PR, but maybe it's not that bad either. It is mirrored on the mailing list, and the only real problem in that case is that maybe the subject is bad -- it looks like discussions about a particular implementation, but is maybe indeed a discussion about the future direction on some functionality.

So, I dunno. Maybe we can recommend that people start up a new thread on the mailing list, if it turns out that some fundamental aspect of the PR needs to be discussed? I don't think it is a bad advice, but it could be formulated more softly as a suggestion.

**Comment by JesperIRL** [comment]
/open

I still intend to get these changes integrated in some form.

File: src/guide/reviewing-and-sponsoring-a-change.md (hunk 9 above)
Comment (JesperIRL) [reply src/guide/reviewing-and-sponsoring-a-change.md]: This is a good advice. I have updated the text to cover this.
I think we should encourage what we think is the right thing to do, not what people in general does - unless the two are the same of course :-)
There are many places in the guide that states that the mailing lists are the right place for discussions since that's what most OpenJDK developers think is the right thing to do.

File: src/guide/reviewing-and-sponsoring-a-change.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+
+If you are new to an area, reviewing changes is also a great way to learn the code and see what general styles and types of changes that are relevant in the area. Be mindful though, if you don't know the area well you should state this in your review comments. As a **R**eviewer you have a responsibility to make sure changes are sound and aline with the general direction of the area. If you, as a **R**eviever, review a change in an area that you don't know well you probably shouldn't be the one to approve the change.
+
+New developers in the OpenJDK community don't have the permissions needed to push changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get their first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually integrate the sponsored contribution into the repository.
```
[the remark this answers is not in the document]
Comment (JesperIRL) [reply src/guide/reviewing-and-sponsoring-a-change.md]: Fixed.

File: src/guide/introduction.md [hunk 15]
```diff
@@ -45,7 +45,7 @@ The rules of any particular project may have different guidelines regarding requ
...
...eight has been seen as a formal lower limit on the number of changes, but since the changes must be non-trivial, or "significant" as the [OpenJDK Project description](https://openjdk.java.net/projects/) says, and the definition of significant is subjective, the general recommendation is to wait with a Committer nomination until there's at least 10-12 changes pushed to have some margin for different interpretations of "significant". In practice though, we have seen several examples where the number of significant changes hasn't been the dominating factor in a Committer vote. A Contributor's work in another OpenJDK project may also be relevant for the vote. What the vote should ultimately test is the Contributor's commitment to the OpenJDK project for which the vote applies - is it believed that the person is dedicated and willing to spend time and effort on the project? Is the person believed to be a good citizen of the project? It's always a good idea to seek the advice of a sponsor who can guide you through the process to becoming a Committer - you will need one to run the Committer vote anyway. They will probably also have a better idea of what constitutes a "significant" change.
```
[the remark this answers is not in the document]
Comment (irisclark) [reply src/guide/introduction.md:48]: Suggest that you `s|openjdk.java.net|openjdk.org|g` to remove all remove all references to the old domain.

File: src/guide/contributing-to-an-open-jdk-project.md [hunk 16]
```diff
@@ -96,6 +98,10 @@ Java and the JDK are very popular products, and just about every Java developer
 
 For reasons like these it’s quite possible that your change, even though it adds value to you, isn’t deemed to add enough value to the larger community.
 
+If you're relatively new to the Java platform then we recommend that you gain more experience writing Java applications before you attempt to work on the JDK itself. The purpose of the sponsored-contribution process is to bring developers who already have the skills required to work on the JDK into the existing development community. The members of that community have neither the time nor the patience required to teach basic Java programming skills or platform implementation techniques.
```
Comment (irisclark) [inline src/guide/contributing-to-an-open-jdk-project.md:101]: "members of that community" -> "members of (the|this) community"

File: src/guide/contributing-to-an-open-jdk-project.md [hunk 17]
```diff
@@ -46,10 +46,12 @@ In many GitHub projects the standard way to propose a change is to create a pull
...
+Like many other open-source communities, the OpenJDK Community requires Contributors to jointly assign their copyright on contributed code. Oracle is the steward of OpenJDK and if you haven't yet signed the [Oracle Contributor Agreement](https://oca.opensource.oracle.com/) (OCA), and are not covered by a company-level agreement, then please do so. This is required in order to make your patch available for review. The OCA gives Oracle and you as a Contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle.
 
 When you sign the OCA, please make sure that you specify your GitHub user name in the `Username` field of the OCA. If you try to create a PR before you have signed the OCA, or if you didn't specify your GitHub user name, you'll get instructions telling you to do so, and the PR won't be published until this is done. OCA registration is a manual process. Please allow for up to several days to have your OCA application processed, even though it's normally processed swiftly. An alphabetical list of all of the assigned OpenJDK usernames may be found on the [OpenJDK people](https://db.openjdk.org/people) list.
```
Comment (irisclark) [inline src/guide/contributing-to-an-open-jdk-project.md:51]: Signing the OCA doesn't get you an OpenJDK username, that won't happen until you acquire a role in a Project or Group.

Perhaps you should provide a link to the [OCA Signatories List][https://oca.opensource.oracle.com/?ojr=contrib-list].

The [OpenJDK Legal Documents][https://openjdk.org/legal/] page contains references to the various relevant legal documents.

**Review by irisclark — APPROVED** [review]
Nice addition to the Guide.

File: src/guide/introduction.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (JesperIRL) [reply src/guide/introduction.md:48]: If it's ok I'll do that in a separate change.

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 17 above)
Comment (JesperIRL) [reply src/guide/contributing-to-an-open-jdk-project.md:51]: Fixed.

File: src/guide/contributing-to-an-open-jdk-project.md (hunk 16 above)
Comment (JesperIRL) [reply src/guide/contributing-to-an-open-jdk-project.md:101]: Fixed.
